gcsweb/Makefile

```diff
@@ -148,3 +134,10 @@ container-clean:
 
 bin-clean:
 	rm -rf .go bin
+
+all-push: all-push-images push-manifest
+
+push-manifest:
+	docker manifest create --amend $(IMAGE):$(VERSION) $(shell echo $(ALL_ARCH) | sed -e "s~[^ ]*~$(IMAGE)\-&:$(VERSION)~g")
```

Comment:
docker manifest has bugs in some versions, which produce invalid manifests.  Except docker proper didn;t validate manifests until recently.  But containerd does...

Reply:
We've been using this flow in k/k for a while without issue, though we do have guards over there in a few places to ensure that you're using docker 18.06.0 or newer (where the bug was fixed).

Google workstations all have new enough docker that I'm not super worried about this.